runtime/compiler/runtime/JITServerAOTCache.hpp

```diff
@@ -403,7 +403,8 @@ class JITServerAOTCacheMap
    JITServerAOTCacheMap();
    ~JITServerAOTCacheMap();
 
-   JITServerAOTCache *get(const std::string &name, uint64_t clientUID);
+   JITServerAOTCache *get(const std::string &name, uint64_t clientUID, bool &pending);
+   void loadAOTCacheFromDisk();
```

Comment:
Nitpick: "from disk" (here and in other places) sounds a bit archaic, "from file" might be better. Also since this method takes no arguments, it's worth adding a comment (or reflect that in the method name) what exact cache we're loading - the next one from the queue.